WalletWasabi/WabiSabi/Backend/Rounds/Round.cs

```diff
@@ -18,15 +16,19 @@ public Round(RoundParameters roundParameters)
 		{
 			RoundParameters = roundParameters;
 
-			AmountCredentialIssuer = new(new(RoundParameters.Random), RoundParameters.Random, MaxRegistrableAmount);
-			VsizeCredentialIssuer = new(new(RoundParameters.Random), RoundParameters.Random, PerAliceVsizeAllocation);
-			AmountCredentialIssuerParameters = AmountCredentialIssuer.CredentialIssuerSecretKey.ComputeCredentialIssuerParameters();
-			VsizeCredentialIssuerParameters = VsizeCredentialIssuer.CredentialIssuerSecretKey.ComputeCredentialIssuerParameters();
-
 			var allowedAmounts = new MoneyRange(roundParameters.MinRegistrableAmount, RoundParameters.MaxRegistrableAmount);
 			var txParams = new MultipartyTransactionParameters(roundParameters.FeeRate, allowedAmounts, allowedAmounts, roundParameters.Network);
 			CoinjoinState = new ConstructionState(txParams);
 
+			InitialInputVsizeAllocation = CoinjoinState.Parameters.MaxTransactionSize - MultipartyTransactionParameters.SharedOverhead;
+			MaxRegistrableVsize = InitialInputVsizeAllocation / RoundParameters.MaxInputCountByRound;
+			MaxVsizeAllocationPerAlice = MaxRegistrableVsize;
```

Comment:
minor nit: it's not a maximum allocation, it's just the allocation. don't bother fixing, there's many names that could be clearer, i think it's going to be easier to just do a call and a review session where we can decide on some new names for the confusing/inaccurate ones.